input/pagecontent/contact.md

```diff
@@ -0,0 +1,3 @@
+### Contact information
+TODO: Add some information about whom to contact in regard to this model
+NOTE: There may be different roles to be filled with different contact details
```

Comment:
I am comparing the organization with https://docs.google.com/document/d/1LrnWmFjG_XuA9l_7u9iUM2S4LyF_7BvZ7dZHmDb8ni0/edit?pli=1#

In the doc, Contact is 3.4 while in the IG, it's 4.